openshift/3scale_backend.conf

```diff
@@ -19,11 +19,13 @@ ThreeScale::Backend.configure do |config|
   config.internal_api.password = "#{ENV['CONFIG_INTERNAL_API_PASSWORD']}"
   config.queues.master_name = "#{ENV['CONFIG_QUEUES_MASTER_NAME']}"
   config.queues.sentinels = "#{ENV['CONFIG_QUEUES_SENTINEL_HOSTS'] && !ENV['CONFIG_QUEUES_SENTINEL_HOSTS'].empty? ? ENV['CONFIG_QUEUES_SENTINEL_HOSTS'] : ENV['SENTINEL_HOSTS']}"
+  config.queues.role = "#{ENV['CONFIG_QUEUES_SENTINEL_ROLE']}".to_sym
```

Comment:
maybe just assume we will get a `String` or `Symbol` here and apply `to_sym` in the code instead of here?

Reply:
I agree, @unleashed. Otherwise, we'll be forcing all the users who create a config file like this to remember to call `to_sym`.

Reply:
https://github.com/3scale/apisonator/pull/1#discussion_r174224359

Reply:
:+1: